sinkingship.py

```diff
@@ -71,7 +72,7 @@ def iterator():
     return tqdm.tqdm(iterator(), total=liked_count)
 
 
-def get_all_posts(*, blog_identifier, api_key):
+def get_all_posts(*, blog_identifier, api_key, before=0):
```

Comment:
minor: You’ve already set the default with this parameter in `@click.option()`, so we don't need to default it to 0 again. Let's enforce passing the parameter here.